texts/pride_prejudice.txt

```diff
@@ -0,0 +1,1833 @@
+"<It>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<is>"
+	"be" vbser pres p3 sg
+"<a>"
+	"a" det ind sg
+"<truth>"
+	"truth" n sg
+"<universally>"
+	"universally" adv
+"<acknowledged>"
+;	"acknowledge" vblex past
+	"acknowledge" vblex pp
+"<,>"
+	"," cm
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<a>"
+	"a" det ind sg
+"<single>"
+	"single" adj
+;	"single" n sg
+"<man>"
+	"man" n sg
+"<in>"
+;	"in" adv
+	"in" pr
+"<possession>"
+	"possession" n sg
+"<of>"
+	"of" pr
+"<a>"
+	"a" det ind sg
+"<good fortune>"
+	"good fortune" n sg
+"<,>"
+	"," cm
+"<must>"
+;	"must" n sg
+;	"must" vbmod inf
+	"must" vbmod pres
+"<be>"
+	"be" vbser inf
+"<in>"
+;	"in" adv
+	"in" pr
+"<want>"
+	"want" n sg
+;	"want" vblex inf
+;	"want" vblex pres
+;	"want" vblex imp
+"<of>"
+	"of" pr
+"<a>"
+	"a" det ind sg
+"<wife>"
+	"wife" n sg
+"<..>"
+	".." sent
+
+
+      
+"<However>"
+	"however" adv
+"<little>"
+;	"little" adj
+	"little" adv
+;	"little" det qnt sg
+"<known>"
+	"known" adj
+;	"know" vblex pp
+"<the>"
+	"the" det def sp
+"<feelings>"
+	"feeling" n pl
+"<or>"
+	"or" cnjcoo
+"<views>"
+	"view" n pl
+;	"view" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<such a>"
+	"such a" det ind sg
+"<man>"
+	"man" n sg
+"<may>"
+;	"may" vbmod inf
+	"may" vbmod pres
+"<be>"
+	"be" vbser inf     
+"<on>"
+;	"on" adv
+	"on" pr
+"<his>"
+;	"his" adj pos pred
+;	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+	"prpers" det pos p3 m sg
+"<first>"
+;	"first" adv
+	"first" adj ord
+"<entering>"
+;	"entering" n sg
+;	"enter" vblex pprs
+;	"enter" vblex subs
+	"enter" vblex ger
+"<a>"
+	"a" det ind sg
+"<neighbourhood>"
+	"neighbourhood" n sg
+"<,>"
+	"," cm
+"<this>"
+	"this" det dem sg
+;	"this" prn dem mf sg
+;	"this" prn rel an mf sp
+"<truth>"
+	"truth" n sg
+"<is>"
+	"be" vbser pres p3 sg
+"<so>"
+;	"so" adv
+	"so" preadv
+;	"so" cnjadv
+"<well>"
+	"well" adv
+;	"well" n sg
+"<fixed>"
+;	"fix" vblex pp
+;	"fix" vblex past
+	"fixed" adj
+;	"fixed" adv
+"<in>"
+;	"in" adv
+	"in" pr
+"<the>"
+	"the" det def sp
+"<minds>"
+	"mind" n pl
+;	"mind" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<the>"
+	"the" det def sp
+"<surrounding>"
+	"surrounding" adj
+;	"surround" vblex pprs
+;	"surround" vblex subs
+;	"surround" vblex ger
+"<families>"
+	"family" n pl
+"<,>"
+	"," cm
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<he>"
+	"prpers" prn subj p3 m sg
+"<is>"
+	"be" vbser pres p3 sg
+"<considered>"
+	"consider" vblex pp
+;	"consider" vblex past
+"<the>"
+	"the" det def sp
+"<rightful>"
+	"*rightful"
+"<property>"
+	"property" n sg
+"<of>"
+	"of" pr
+"<some>"
+;	"some" adj
+	"some" adv
+;	"some" det qnt sp
+;	"some" prn ind mf pl
```

Comment:
not sure of this one, maybe should be `<det>`?

Reply:
Yes, you're right, since it is determining a pronoun next